### PR TITLE
Add warmup EC2 agent TeamCity job configuration

### PR DESCRIPTION
**Pull request by guylabs**

### CODE REVIEW

File: .teamcity/Gradle_Util_Performance/settings.kts [hunk 1]
```diff
@@ -32,5 +32,5 @@ node (Plugins -> teamcity-configs -> teamcity-configs:generate),
 the 'Debug' option is available in the context menu for the task.
 */
 
-version = "2019.1"
-project(Gradle_Util_Performance.Project)
+version = "2019.2"
+project(Project)
```
Comment (donat) [inline .teamcity/Gradle_Util_Performance/settings.kts]: 👍

File: .teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package Gradle_Util.buildTypes
+
+import common.gradleWrapper
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.Requirement
+import jetbrains.buildServer.configs.kotlin.v2019_2.RequirementType
+
+object WarmupEc2Agent : BuildType({
+    uuid = "980bce31-2a3e-4563-9717-7c03e184f4a4"
```
Comment (donat) [inline .teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt]: Nit: I don't remember using this format for UUIDs. IIRC the uuid is the project id prefixed. Could you please look it up.

File: .teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    name = "Warmup EC2 Agent"
+    id("Gradle_Util_WarmupEc2Agent")
```
Comment (wolfs) [inline .teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt]: I suppose you also want to attach a VCS root to the build, right?

File: .teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt (hunk 2 above)
Comment (wolfs) [reply .teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt]: I think the format of the uuid doesn't really matter, as long as it doesn't change over time.

File: .teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt (hunk 3 above)
Comment (guylabs) [reply .teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt]: Yes, but isn't that already set by default? How does it then work on the _Util/Performance_ builds as there I do not see any specific VCS root attached.

Comment (guylabs) [reply .teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt]: Thanks for the hint. Done https://github.com/gradle/gradle/commit/63f6128a1d1d6a3f5b2b1ceb22066a1847748ee0

**Pull request merged**